src/server/server.ts

```diff
@@ -283,7 +273,7 @@ export class SubscriptionServer {
 
   private sendSubscriptionSuccess(connection: WebSocket, subId: string): void {
```

Comment:
there is no real need for success message, if there request failed, the user will get an error message.
if it success, it will start stream data.